crates/iota-node/src/lib.rs

```diff
@@ -2067,16 +2066,9 @@ pub async fn build_http_server(
             ))?;
         }
 
-        let iota_names_config = match &config.iota_names_config {
-            Some(config) => config.clone(),
-            None => {
-                let chain = state
-                    .get_chain_identifier()
-                    .ok_or(IotaError::Storage("chain identifier not found".to_string()))?
-                    .chain();
-                IotaNamesConfig::from_chain(&chain)
-            }
-        };
+        // TODO: Init from chain if config is not set when `IotaNamesConfig::from_chain`
+        // once implemented
```

Comment:
So we revert this when Names is deployed on both Testnet and Mainnet?

Reply:
I think it would make sense to revert this, so the node can run automatically with the chain configuration for `devnet/testnet/mainnet` - upstream is doing the same.